src/schema/v2/fields/money.ts

```diff
@@ -50,11 +51,17 @@ export const amount = (centsResolver) => ({
       type: GraphQLString,
       defaultValue: ",",
     },
+    disambiguate: {
+      type: GraphQLBoolean,
+      defaultValue: false,
+    },
```

Comment:
I like that we're aiming for the real goal here, which is to disambiguate ambiguous symbols. This aligns with the [ruby money gem](https://github.com/RubyMoney/money/blob/2d9153b02257783d75a5f3fe9b80bf8250bc67be/lib/money/money/formatter.rb#L175-L182) that MP's [source of truth](https://github.com/artsy/metaphysics/blob/c4763ac50a5b25143723f618a489f5c1ad40eed5/src/lib/currency_codes.json) is based on. This also helps push us towards more consistent display across surface areas, vs. supporting every arbitrary formatting.

Reply:
Unrelated to this PR, I'd be interested in consolidating the source of truth currency_code.json (which was [copied over from the money gem](https://github.com/artsy/metaphysics/pull/883) and has been diverged from that) with the gem. Potentially we can try similar approach in [address_helper](https://github.com/artsy/address_helper/pull/1), and move the file to the [money_helper](https://github.com/artsy/money_helper) gem as the single place. I might try to spike on that on a #FF.